src/main/app/src/app/zaken/zaak-create/zaak-create.component.ts

```diff
@@ -115,8 +136,17 @@ export class ZaakCreateComponent implements OnInit {
         if (this.acties.toevoegenInitiatorPersoon || this.acties.toevoegenInitiatorBedrijf) {
             zaaktypeEnInitiator.push(this.initiatorField);
         }
-        this.createZaakFields = [[titel], zaaktypeEnInitiator, [startdatum, registratiedatum, this.locatieField], [tussenTitel],
-            [communicatiekanaal, vertrouwelijkheidaanduiding], [omschrijving], [toelichting]];
+        this.createZaakFields = [
+            [titel],
+            zaaktypeEnInitiator,
+            [startdatum, this.locatieField],
+            [toekennenGegevensTitel],
+            [this.medewerkerGroepFormField],
+            [overigeGegevensTitel],
```

Comment:
Heb je dit overlegd met Robert? Zelf zou ik denken dat het toekennen aan de medewerker en/of groep de laatste stap is voor iemand die een zaak aanmaakt. Zodat het dan dus onderaan het best op z'n plek is waardoor je dit Overig-label (wat natuurlijk functioneel vaag is) niet nodig hebt. Maar overleg dat even...

Reply:
Is overlegd